components/console_simple_init/console_simple_init.c

```diff
@@ -67,6 +68,34 @@ esp_err_t console_cmd_user_register(char *cmd, esp_console_cmd_func_t do_user_cm
     return ret;
 }
 
+
+/**
+ * @brief Register all the console commands in .plugins_desc section
+ *
+ * @return
+ *          - esp_err_t
+ */
+esp_err_t console_cmd_all_register(void)
+{
+    esp_err_t ret = ESP_FAIL;
+    extern const console_cmd_plugin_desc_t _plugins_array_start;
+    extern const console_cmd_plugin_desc_t _plugins_array_end;
+
+    printf("List of plugin commands:\n");
```

Comment:
It looks like you have copy-pasted IDF example code here. Is logging really necessary in this case? If it is, could we use the standard ESP_LOG mechanism instead of writing directly to stdout?

Reply:
It's reassuring to see the list of all registered plugin components at the starting of the application.
Change the `printf` to `ESP_LOGI`.